torch/_inductor/codecache.py

```diff
@@ -2804,18 +2879,18 @@ def wait(self, scope: Dict[str, Any]) -> None:
             for key, result in scope.items():
                 if config.verbose_progress and not isinstance(pbar, _Faketqdm):
                     pbar.set_postfix_str(key)
-                if isinstance(result, (Future, TritonFuture)):
+                if isinstance(result, (Future, CodeCacheFuture)):
                     scope[key] = result.result()
                     pbar.update(1)
 
         _compile_end()
 
 
-if os.environ.get("TORCH_TNT_IN_USE", "0") == "1":
-    # When TorchTNT is used, calling warm_pool() here will cause the
-    # compile workers created not being able to be shut down inside
-    # shutdown_compile_workers(). This may cause significant QPS drop.
-    log.info("Do not call AsyncCompile.warm_pool() because TorchTNT is in use.")
```

Comment:
Should we keep this `log.info`, or is it fixed by this PR?

Reply:
Doesn't seem that useful to me